tests/test_sitemap.py

```diff
@@ -17,22 +18,59 @@
 class SitemapTest(BaseTest):
 
     def test_sitemap_base(self):
-        test_string = '<url><loc>http://example.com/it/</loc><lastmod>%s</lastmod><changefreq>monthly</changefreq><priority>0.5</priority></url>' % now().strftime('%Y-%m-%d')
-        self.get_pages()
+        page1, page2, page3 = self.get_pages()
+        if is_versioning_enabled():
+            page1_content = create_title(title='pagecontent1', language='en', page=page1,
```

Comment:
get_pages already creates the page contents. I would advise doing the publishing in one place i.e. get_pages, this is how it used ot work: https://github.com/nephila/djangocms-page-sitemap/blob/develop/tests/base.py#L37